tests/release/e2e-test-releaseplan-owner-ref.go

```diff
@@ -42,6 +42,7 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-2469]test-releaseplan-owner-ref-a
 		It("verifies that the ReleasePlan has an owner reference for the application", func() {
 			releasePlan, err = fw.AsKubeAdmin.ReleaseController.GetReleasePlan(sourceReleasePlanName, devNamespace)
 			Expect(err).NotTo(HaveOccurred())
+			Expect(len(releasePlan.OwnerReferences)).To(BeNumerically(">", 0))
```

Comment:
thus I should implement  the test with  `Eventually()` waiting that the owner reference be there ,  if it waited and still didn't find then will fail test  
@davidmogar wdyt?

Reply:
It sounds more reasonable.

Reply:
I like the Eventually assertion better as well. But is this something that would take a very long to wait for?

Reply:
After @davidmogar explained to about process of updating the releasePlan.OwnerReferences , 
so we set the timeout to 1 min, since it's about reconciling duration .